tests/serializers/test_data.py

```diff
@@ -239,29 +240,28 @@ def inherited_compare(testcase, pk, klass, data):
         testcase.assertEqual(value, getattr(instance, key))
 
 
-# Define some data types. Each data type is
-# actually a pair of functions; one to create
-# and one to compare objects of that type
-data_obj = (data_create, data_compare)
-generic_obj = (generic_create, generic_compare)
-fk_obj = (fk_create, fk_compare)
-m2m_obj = (m2m_create, m2m_compare)
-im2m_obj = (im2m_create, im2m_compare)
-im_obj = (im_create, im_compare)
-o2o_obj = (o2o_create, o2o_compare)
-pk_obj = (pk_create, pk_compare)
-inherited_obj = (inherited_create, inherited_compare)
+# Define some data managers. Each manager has a pair of functions;
+# one to create and one to compare objects of a particular type.
+Manager = namedtuple("Manager", ["create_object", "compare_objects"])
+data_obj = Manager(data_create, data_compare)
+generic_obj = Manager(generic_create, generic_compare)
+fk_obj = Manager(fk_create, fk_compare)
+m2m_obj = Manager(m2m_create, m2m_compare)
+im2m_obj = Manager(im2m_create, im2m_compare)
+im_obj = Manager(im_create, im_compare)
+o2o_obj = Manager(o2o_create, o2o_compare)
+pk_obj = Manager(pk_create, pk_compare)
+inherited_obj = Manager(inherited_create, inherited_compare)
```

Comment:
I added a named tuple here to do `manager.create_object` instead of `func[0]` etc (see `assert_serializer`)
I kept the naming of `data_obj` etc to reduce the git diff.

Reply:
I like it, it really adds readability and the cost is minimal. I say we keep it 🙋🏻

Reply:
Do we ever use `Manager` as a `tuple` now? If not, maybe a `dataclass` would be slightly cleaner still?

Reply:
I also think this is an improvement!

Reply:
> Do we ever use Manager as a tuple now? If not, maybe a dataclass would be slightly cleaner still?

We don't use it as a tuple. However I only see one `dataclass` in the Django codebase which makes me a little suspicious of adding it. We are also avoid adding types so maybe...
```
@dataclass
class Manager:
    create_object: ...
    compare_objects: ...
```

This is not something I feel very strongly about so happy for suggestions :+1:

Reply:
You can swap out `namedtuple` with `make_dataclass` (`from data classes import make_dataclass`), but at that point I don't really see why that's an improvement 🤔

Reply:
My main reason was to avoid the tuple behaviour, but it's a very minor point.